app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -67,6 +67,15 @@ define([
       return ! account.has('profileImageUrl');
     },
 
+    _metricsImageChange: function (account) {
+      // if the user already has an image set, then report a change event
+      if (account.has('profileImageId')) {
+        this.logScreenEvent('submit.change');
```

Comment:
Can you add these new events to [client-metrics.md](https://github.com/mozilla/fxa-content-server/blob/master/docs/client-metrics.md) under the appropriate screens?